Sources/SQLKit/Builders/SQLCaseBuilder.swift

```diff
@@ -75,3 +75,16 @@ extension SQLCaseBuilder {
         return self.else(SQLBind(value))
     }
 }
+
+
+extension SQLSelectBuilder {
+    public func column(`case`: (SQLCaseBuilder) -> SQLCaseBuilder) -> Self {
+        return self.column(`case`(.init()).query)
```

Comment:
@calebkleveter Sorry, I did write it the way I originally had it for a reason - it's not technically safe to rely on the closure the user provides returning the same builder.

```suggestion
        let builder = SQLCaseBuilder()
        _ = `case`(builder)
        return self.column(builder.query)
```